stress/src/throughput.rs

```diff
@@ -79,12 +78,14 @@ where
         let mut system = System::new_all();
 
         loop {
-            let elapsed = end_time.duration_since(start_time).as_secs();
+            let current_time = Instant::now();
+            let elapsed = current_time.duration_since(last_collect_time).as_secs();
             if elapsed >= SLIDING_WINDOW_SIZE {
```

Comment:
Is this check still needed? This seems always true because the 5 secnods sleep is included.

Reply:
yes, it's unclear to me too why we sleep more than SLIDING_WINDOW_SIZE. Can possibly be other way.

Reply:
There are quite a few simplifications that can be made to the Stress Test. I intend to cover them in a separate PR and just fix the calculation error in this PR for easier review. Would you want me to include other changes in this PR itself?

Reply:
Thanks, I think this PR is important to get the correct throughput for now. Rest can follow.